internal/backend/schedule/schedule.go

```diff
@@ -0,0 +1,98 @@
+package schedule
+
+import (
+	"context"
+	"fmt"
+
+	"go.temporal.io/sdk/client"
+	"go.uber.org/zap"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/dispatcher"
+	"go.autokitteh.dev/autokitteh/internal/backend/temporalclient"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+type temporalScheduleImpl struct {
+	z      *zap.Logger
+	tmprl  temporalclient.Client
+	events sdkservices.Events
+}
+
+type TemporalSchedule interface {
+	CreateScheduledWorkflow(ctx context.Context, scheduleID string, schedule string, triggerID sdktypes.TriggerID) error
+	DeleteSchedulerWorkflow(ctx context.Context, scheduleID string) error
+	UpdateSchedulerWorkflow(ctx context.Context, scheduleID string, schedule string) error
+}
+
+func New(z *zap.Logger, events sdkservices.Events, tc temporalclient.Client) TemporalSchedule {
+	return &temporalScheduleImpl{z: z, events: events, tmprl: tc}
+}
+
+func (tsc *temporalScheduleImpl) CreateScheduledWorkflow(ctx context.Context, scheduleID string, schedule string, triggerID sdktypes.TriggerID) error {
+	event := kittehs.Must1(sdktypes.EventFromProto(&sdktypes.EventPB{EventType: sdktypes.SchedulerEventTriggerType}))
+	eventID, err := tsc.events.Save(ctx, event)
+	if err != nil {
+		return fmt.Errorf("create scheduler workflow: save event: %w", err)
+	}
+
+	z := tsc.z.With(zap.String("event_id", eventID.String()))
+	z.Debug("create scheduled event", zap.String("schedule", schedule))
+
+	_, err = tsc.tmprl.ScheduleClient().Create(ctx,
+		client.ScheduleOptions{
+			ID: scheduleID,
+			Spec: client.ScheduleSpec{
+				CronExpressions: []string{schedule},
```

Comment:
do we want to allow multiple schedules on the same trigger as well?

Reply:
not for now I think. If there will be a request we could implement it. Meanwhile it supports everything robfig/cron
- standard cron expression
- extended cron expression (up to 7 *, including seconds)
- @monthly, @daily etc shortcuts
- @every <duration>